### ISSUE TITLE
Review 508 onresource page & get approval 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#1908]
https://onrresourcestage/508-Compliance/

**Comment by mcharg** [comment]
I sent the following feedback to Justin:

- There's a missing link for digital signatures. I sent him the correct link.
- Copy about which PPT template to use was missing.
- He's going to make all external links open in a new tab. They're currently a mix of new tab vs. not.

Everything else looks good.

**Comment by mcharg** [comment]
@Maroyafaied Justin put in all the changes. Do you have any additional feedback?
https://onrresourcestage/508-Compliance/

**Comment by Maroyafaied** [comment]
@mcharg Would it be possible for Justin to add the page on the main menu navigation before I send it for review?

**Comment by mcharg** [comment]
@Maroyafaied I think he said it would end up on the live site if he did that, so he was going to wait until it's approved.

**Comment by Maroyafaied** [comment]
Sent email to Howard
